birdhouse/components/weaver/config/canarie-api/weaver_config.py.template

```diff
@@ -0,0 +1,9 @@
+SERVICES['node']['monitoring'].update({
```

Comment:
@fmigneault another problem detected after go-live in prod yesterday:

This `weaver_config.py` should be added to `.gitignore`, else autodeploy is broken because it would detect an unclean checkout.

Reply:
Do you want a separate PR for this fix only (with version bump and all?).